CONTRIBUTING.md

```diff
@@ -1,30 +1,29 @@
 # Contributing
 
-Galaxy welcomes new development!
-This document briefly describes how to contribute to the [core
-galaxy project](https://github.com/galaxyproject/galaxy) -
-also checkout our 2013 Galaxy Community
-Conference presentation on the topic
+Galaxy welcomes new development!  This document briefly describes how to
+contribute to the [core galaxy
+project](https://github.com/galaxyproject/galaxy) - also checkout our 2013
+Galaxy Community Conference presentation on the topic
 ([video](https://vimeo.com/channels/581875/73486255),
-[presentation](https://depot.galaxyproject.org/hub/attachments/documents/presentations/gcc2013/BakerContribute.pdf)). For
-information on contributing more broadly to the Galaxy ecosystem and a
+[presentation](https://depot.galaxyproject.org/hub/attachments/documents/presentations/gcc2013/BakerContribute.pdf)).
+For information on contributing more broadly to the Galaxy ecosystem and a
 deeper discussion of some of these points - please see the
-[Develop](https://galaxyproject.org/develop) section of the
-[Galaxy Community Hub](https://galaxyproject.org).
+[Develop](https://galaxyproject.org/develop) section of the [Galaxy Community
+Hub](https://galaxyproject.org).
 
 ## Before you Begin
 
-If you have an idea for a feature to add or an approach for a bugfix,
-it is best to communicate with Galaxy developers early. The most
-common venues for this are
-[GitHub issues](https://github.com/galaxyproject/galaxy/issues).
-Browse through existing GitHub issues and Trello cards and if one seems related,
-comment on it. We also maintain a [tag](https://github.com/galaxyproject/galaxy/issues?q=is%3Aissue+is%3Aopen+label%3Afriendliness%2Ffriendly) on Github for
-smaller issues we believe would make the best entry points for new
-developers.
-Galaxy developers are generally available via
-[IRC](https://galaxyproject.org/get-involved), [Gitter](https://gitter.im/galaxyproject/Lobby)
-and on the [development mailing list](http://dev.list.galaxyproject.org).
+If you have an idea for a feature to add or an approach for a bugfix, it is
+best to communicate with Galaxy developers early. The most common venues for
+this are [GitHub issues](https://github.com/galaxyproject/galaxy/issues).
+Browse through existing GitHub issues and Trello cards and if one seems
```

Comment:
Time to also drop the references to Trello?

Reply:
@nsoranzo Sure, one sec.